### ISSUE TITLE
Make merge sort functional

### ISSUE THREAD

**Issue by purvil12c** [purvil12c/Sorting-Visualizer-React#1]
Add merge sort algorithm functionality

**Comment by Anurag26** [comment]
I think this is closed now.

**Comment by purvil12c** [comment]
The implementation for merge sort still remains.

**Comment by Anurag26** [comment]
SortingAlgorithm.js
Line 1-40 shows the code for merge sort.

**Comment by purvil12c** [comment]
That is true, there is a flag on the radio buttons which makes the merge sort disabled, I think while on working on the merge sort there were some issues that I was running into and had to keep that functionality disabled. Will have to dig into it deeper.

**Comment by Anurag26** [comment]
Oh yeah, the numbers disappear.
